nft-book/utils/convertToCsv.js

```diff
@@ -0,0 +1,49 @@
+import { parse } from "json2csv";
+import fs from "fs";
+const basePath = process.cwd();
+
+const readDir = `${basePath}/output/json`;
+const rawData = fs.readFileSync(`${readDir}/_metadata.json`);
+const converted = JSON.parse(rawData);
+
+function shuffle(a, b) {
+  const num = Math.random() > 0.5 ? -1 : 1;
+  return num;
+}
+
+function formatIndex(index) {
+  if (index < 10) {
+    return `000${index}`;
+  }
+  if (index === 10 || index < 100) {
+    return `00${index}`;
+  }
+  if (index === 100 || index < 1000) {
```

Comment:
is the `index === 10` `index === 100` part redundant?

Reply:
🌚